apps/ios/src/index.tsx

```diff
@@ -2,11 +2,18 @@
 
 import {FluentTesterApp} from '@fluentui-react-native/tester';
 import * as React from 'react';
-import {AppRegistry, SafeAreaView} from 'react-native';
+import {AppRegistry, SafeAreaView, Text, useColorScheme} from 'react-native';
 
 const FluentTester: React.FunctionComponent = () => {
+  const isDarkMode = useColorScheme && useColorScheme() === 'dark';
+
+  const backgroundStyle = {
+    backgroundColor: isDarkMode ? 'black' : 'white',
```

Comment:
Note: https://github.com/microsoft/react-native-test-app/pull/286 will remove the need for this once it's checked in and published

Reply:
is it published yet?

Reply:
Yes, I'll just include the new package with this PR

Reply:
Eh... Updating RNTA from 0.3.2 to 0.4.3 works on iOS and macOS but breaks android. With my other changes with FluentTester layout, we don't need to update yet so I'll leave it off this review